src/NodeAnalyzer/TerminatedNodeAnalyzer.php

```diff
@@ -45,7 +46,7 @@ final class TerminatedNodeAnalyzer
      */
     private const ALLOWED_CONTINUE_CURRENT_STMTS = [InlineHTML::class, Nop::class];
 
-    public function isAlwaysTerminated(StmtsAwareInterface $stmtsAware, Stmt $node, Stmt $currentStmt): bool
+    public function isAlwaysTerminated(StmtsAwareInterface|ClassMethod $stmtsAware, Stmt $node, Stmt $currentStmt): bool
```

Comment:
This change is not needed, `ClassMethod` is already instanceof `StmtsAwareInterface`

Reply:
https://github.com/rectorphp/rector-src/pull/6969